src/expectEvent.js

```diff
@@ -138,4 +175,9 @@ function isTruffleContract (contract) {
 expectEvent.inLogs = deprecate(inLogs, 'expectEvent.inLogs() is deprecated. Use expectEvent() instead.');
 expectEvent.inConstruction = inConstruction;
 expectEvent.inTransaction = inTransaction;
+
+expectEvent.not = dontExpectEvent;
```

Comment:
What is the purpose of this function? I would simply store an object here:

```suggestion
expectEvent.not = {};
```

And then remove the definition of `dontExpectEvent` since it isn't used anywhere else.